web/templates/blocks/hosts_table.html.tmpl

```diff
@@ -18,14 +20,10 @@
                     <td>
                         <a href="/clusters/{{ index .TrentoMeta "trento-ha-cluster" }}">{{ index .TrentoMeta "trento-ha-cluster" }}</a>
                     </td>
-                    <td>
-                        {{- range $Key, $Value := .TrentoMeta }}
-                            {{- if eq $Key "trento-ha-cluster" }}
-                            {{- else }}
-                                <span class='badge badge-pill badge-info'>{{ $Value }}</span>
-                            {{- end }}
-                        {{- end }}
-                    </td>
+
+                    <td>{{ .TrentoMetaKey "trento-sap-environment" }}</td>
+                    <td>{{ .TrentoMetaKey "trento-sap-landscape" }}</td>
```

Comment:
I think it would be good to put urls to the specific entries as we do in the `cluster` column, so you can move around easily